README.md

```diff
@@ -100,9 +100,10 @@ require('yabs'):setup({
     },
     optional = {
       command = 'echo runs on condition',
-      -- You can specify a condition which determines whether to enable a
-      -- specific task (it could be your custom function)
-      condition = require('yabs.conditions').file_exists('filename.txt'),
+      -- You can use any function to specify a condition which determines
+      -- whether to enable a specific task
+      -- (here we use Path module from Plenary to check if the file exists)
+      condition = require('plenary.path'):new('filename.txt'):exists(),
```

Comment:
Actually this doesn't result in the same behavior. AFAIK the condition key has to be a function (I don't know what will happen if it's a boolean instead, probably throw an error when it tries to call it like a function), and even if we did add support for it to be a boolean instead, it still wouldn't be the same. As it stands right now, the function will be called every time you run `yabs:run_task`, so it will check for a file every time. If it was just a fixed value (like the one returned from a plenary Path:exists() call), it would always stay the same. So if the file was created after the first call to `yabs:setup`, with a fixed bool, it would still not run until you restart neovim or call `yabs:setup` again.

Come to think of it, adding support for a fixed bool value might be useful, for example if the condition takes a while to run, the user should be able to have it only run once. But even then, this would not be 1-to-1 behavior.

The solution to this would be to either do what I suggested with the `conditions` module or wrap it in a function in the README:

```lua
condition = function() require('plenary.path'):new('filename.txt'):exists() end
```

Reply:
Sounds reasonable, I thought to use `require('plenary.path'):new('filename.txt'):exists` (without braces) but it will check only first time because Path relies on cached stats. Will do as you suggested.